docs/INSTALL.rst

```diff
@@ -395,6 +395,8 @@ former is a WSGI application, which we recommend running with Apache's
 (The file may already exist, with just the ``LoadModule`` option. If so, it is
 safe to replace it.)
 
+**Note:** if accessing HIL through a public IP address, be sure to change the ``VirtualHost`` and ``ServerName`` IP addresses to match the public one.
```

Comment:
I'd suggest making this something more like `be sure to update your VirtualHost entry accordingly`, with a link to the docs at https://httpd.apache.org/docs/current/mod/core.html#virtualhost